### PR TITLE
ScannedMaps should not be start_canvases

### LINKED ISSUE #6456: Scanned Map is not rendering a IIIF Manifest
## Success Criteria

- [ ] https://figgy.princeton.edu/catalog/1419afdd-0766-44f1-8bba-92c251a6885a renders a viewer.

## First Step

Go through the backtrace and find out why it's calling `ocr_content`.

## Backtrace

    line 414 of [PROJECT_ROOT]/app/services/manifest_builder.rb: ocr_content
    line 14 of [PROJECT_ROOT]/app/services/manifest_builder/canvas_rendering_builder.rb: apply
    line 8 of [PROJECT_ROOT]/app/services/manifest_builder/canvas_builder.rb: apply_record_properties

[View full backtrace and more info at honeybadger.io](https://app.honeybadger.io/projects/53391/faults/109045278)

## Sudden Priority Justification

The resource is broken for viewing in a way that our staff can't fix.

### PR DESCRIPTION
**Pull request by tpendragon**
If a scanned map gets set as a start canvas, don't try to render it as one.

Closes #6456

### CODE REVIEW

File: spec/services/manifest_builder/by_resource_type/manifest_builder_scanned_maps_spec.rb [hunk 1]
```diff
@@ -38,7 +38,7 @@
   context "when given a nested scanned map set" do
     subject(:manifest_builder) { described_class.new(query_service.find_by(id: scanned_map.id)) }
     let(:scanned_map) do
-      FactoryBot.create_for_repository(:scanned_map, description: "Test Description", member_ids: child.id)
+      FactoryBot.create_for_repository(:scanned_map, description: "Test Description", member_ids: child.id, start_canvas: child.id)
```
Comment (hackartisan) [inline spec/services/manifest_builder/by_resource_type/manifest_builder_scanned_maps_spec.rb:41]: It's not clear now why the test setup has the start canvas. Maybe update the context to say that it's mistakenly set as a start canvas? I don't know, it might be better to have a separate test so that it's clear this is one is sort of a happy path test, have a new one for this edge case.

**Review by hackartisan — CHANGES_REQUESTED** [review]
The title is ScannedMaps should not be start_canvas but the code reads like, only a file set should be a start_canvas. seems fine, just attempting to add clarity. please see my comment below.

File: spec/services/manifest_builder/by_resource_type/manifest_builder_scanned_maps_spec.rb (hunk 1 above)
Comment (tpendragon) [reply spec/services/manifest_builder/by_resource_type/manifest_builder_scanned_maps_spec.rb]: Good read, made a new one.

**Review by hackartisan — APPROVED** [review]
